packages/compat/package.json

```diff
@@ -16,9 +16,11 @@
   "files": [
     "dist/cjs/index.cjs",
     "dist/cjs/index.d.cts",
+    "dist/cjs/types.ts",
     "dist/cjs/types.d.ts",
     "dist/esm/index.js",
     "dist/esm/index.d.ts",
+    "dist/esm/types.ts",
     "dist/esm/types.d.ts",
     "README.md",
     "LICENSE"
```

Comment:
It looks like we could just include `"dist"` here instead of each single file. BTW, README and LICENSE are included automatically regardless of extension, so it's not required to list them explicitly: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files.

Reply:
Ah good call. I was copy-pasting between `jsr.json` and `package.json`.